### PR TITLE
Website announcements fixes and additions

### PR DESCRIPTION
**Pull request by sudokoko**
This PR is a piggyback off of #810, adding the ability to include line breaks in announcement text as well as displaying which user published the announcement.

### CODE REVIEW

File: ProjectLighthouse.Servers.Website/Pages/AnnouncePage.cshtml [hunk 1]
```diff
@@ -37,11 +37,15 @@
...
-                @announcement.Content
-            </p>
-
+            <div>
+                <h3>@announcement.Title</h3>
+                <div style="padding-bottom: 2em;">
+                    <span style="white-space: pre-line; ">@announcement.Content</span>
+                </div>
+                <div class="ui tiny bottom left attached label">
+                    Posted by <span style="color: black">@announcement.Publisher</span>
```
Comment (Zaprit) [inline ProjectLighthouse.Servers.Website/Pages/AnnouncePage.cshtml:46]: this could have an avatar, and link back to the users profile

**Review by Zaprit — CHANGES_REQUESTED** [review]
See changes, otherwise looks good

File: ProjectLighthouse.Servers.Website/Pages/AnnouncePage.cshtml (hunk 1 above)
Comment (sudokoko) [reply ProjectLighthouse.Servers.Website/Pages/AnnouncePage.cshtml:46]: Avatar messes with the styling, though I'm open to linking back to the user.

**Review by Zaprit — APPROVED** [review]
All looks good

**Review by Zaprit — APPROVED** [review]
Looks good

**Review by Zaprit — APPROVED** [review]
I am forever reviewing this PR, by the time I've written this review there will be another commit

**Review by daggintosh — APPROVED** [review]
How many more changes does Zakrit need to review

**Review by Slendy — APPROVED** [review]
image

**Pull request merged**